dev_guide/deployments/kubernetes_deployments.adoc

```diff
@@ -0,0 +1,187 @@
+      - name: nginx
+        image: nginx:1.7.9
+        ports:
+        - containerPort: 80
+----
+
+You can use the CLI to operate on Kubernetes deployments and replica sets like
+other object types, as described in
+xref:../../cli_reference/basic_cli_operations.adoc#oc-common-operations[Common Operations], like `create`, `get`, and `describe`. Use `deployments` for
+Kubernetes deployments and `replicasets` or `rs` for replica sets as the object
+type.
+
+See the Kubernetes documentation for more details about
+link:http://kubernetes.io/docs/user-guide/deployments/[Deployments] and
+link:http://kubernetes.io/docs/user-guide/replicasets/[Replica Sets],
+substituting `oc` for `kubectl` in CLI usage examples.
+
+[[deployments-vs-deployment-configurations]]
+== Deployments vs Deployment Configurations
+
+Because deployment configurations existed in {product-title} prior to
+deployments being added in Kubernetes 1.2, the latter object type naturally
+diverges slightly from the former. The long-term goal in {product-title} is to reach
+full feature parity in Kubernetes deployments and switch to using them as a
+single object type that provides fine-grained management over applications.
+
+Kubernetes deployments are supported to ensure upstream projects and examples
+that use the new object type can run smoothly on {product-title}. Given the
+current feature set of Kubernetes deployments, you may want to use them instead
+of deployment configurations in {product-title} if you do not plan to use any of
+the following in particular:
+
+- xref:../../dev_guide/managing_images.adoc#dev-guide-managing-images[image streams]
+- xref:../../dev_guide/deployments/deployment_strategies.html#lifecycle-hooks[lifecycle hooks]
+- xref:../../dev_guide/deployments/deployment_strategies.html#custom-strategy[Custom deployment strategies]
+
+The following sections go into more details on the differences between the two
+object types to further help you decide when you might want to use Kubernetes
+deployments over deployment configurations.
+
+[[deployment-configuration-specific-features]]
+=== Deployment Configuration-Specific Features
+
+[[dc-vs-d-automatic-rollbacks]]
+==== Automatic Rollbacks
+
+Kubernetes deployments do not support automatically rolling back to the last
+successfully deployed replica set in case of a failure. This feature should be
+added soon.
+
+[[dc-vs-d-triggers]]
+==== Triggers
+
+Kubernetes deployments have an implicit `ConfigChange` trigger in that every
+change in the pod template of a deployment automatically triggers a new rollout.
+If you do not want new rollouts on pod template changes, pause the deployment:
+
+----
+$ oc rollout pause deployments/<name>
+----
+
+At the moment, upstream Deployment do not support ImageChange triggers. A generic triggering
+mechanism has been proposed upstream but it is unknown if and when will it be accepted.
+Eventually, we can implement our own mechanism on top but it would be more desirable for it
```

Comment:
maybe reword this sentence to focus on "you" instead of "we". Also, the "on top" is confusing. Perhaps reword to:

Eventually, you will be able to implement your own mechanism. However, it is more desirable to have this exist as part of the Kubernetes core.